### PR TITLE
Improve full text searching for titles, contributors, etc

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: config/initializers/spotlight_initializer.rb [hunk 1]
```diff
@@ -24,7 +24,7 @@
 # Spotlight::Engine.config.solr_batch_size = 20
 # Spotlight::Engine.config.filter_resources_by_exhibit = true
 # Spotlight::Engine.config.autocomplete_search_field = 'autocomplete'
-# Spotlight::Engine.config.default_autocomplete_params = { qf: 'id^1000 full_title_tesim^100 id_ng full_title_ng' }
+Spotlight::Engine.config.default_autocomplete_params = { qf: 'id^1000 full_title_tsim^100 id_ng full_title_ng' }
```
Comment (jcoyne) [inline config/initializers/spotlight_initializer.rb]: Should this be `cho_title_tsim`?

File: solr/config/schema.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
     <field name="lng" type="tdouble" stored="true" indexed="true" multiValued="false"/>
     <field name="original_pid_tesim" type="pid_text" stored="true" indexed="true" multiValued="true"/>
 
-    <field name="full_title_ng" type="text_en_ng" stored="false" indexed="true" multiValued="true"/>
+    <field name="cho_title_ng" type="text_en_ng" stored="false" indexed="true" multiValued="true"/>
```
Comment (jcoyne) [inline solr/config/schema.xml]: Some of our titles are non-english, this won't cause a problem, will it?

File: config/initializers/spotlight_initializer.rb (hunk 1 above)
Comment (cbeer) [inline config/initializers/spotlight_initializer.rb]: thanks

File: solr/config/schema.xml (hunk 2 above)
Comment (cbeer) [inline solr/config/schema.xml]: I don't /think/ so. This is just an ngrams field to support autocomplete in sir trevor widgets, so a little slop here and there oughtn't make a big difference.

**Pull request merged**